engine/_classes/class.bertagallery.php

```diff
@@ -135,7 +131,9 @@ public static function getHTML($imgs, $mediaFolderName, $galleryType, $isAdminMo
                     $spaceBetweenItems = 12;
                     $totalWidth += $itemWidth + $spaceBetweenItems;
 
-                    $galleryContent .= $itemHTML;
+                    if ($i < $imageLimit) {
```

Comment:
why did you change the if statement? Wasn't it better before?

Reply:
Because we can't break the loop as before. We need to loop through all the items to get the total width of gallery.

Reply:
Also doesn't the navigation part already loop over all the images?

Reply:
No, it generates only the limited images, see
```
if ($i < $imageLimit) {
  $galleryContent .= $itemHTML;
}
```

all the other images are loaded in frontend based on the gallery navigation html code